components/automate-gateway/api/iam/v2/teams.proto

```diff
@@ -58,16 +174,68 @@ service Teams {
     option (chef.automate.api.iam.policy).resource = "iam:teams:{id}";
     option (chef.automate.api.iam.policy).action = "iam:teams:delete";
   };
+
+  /*
+  Gets local team membership
+
+  Lists all users of a local team. Users are listed by their membership_id.
+
+  Authorization Action:
+  ```
+  iam:teamUsers:list
+  ```
+  */
   rpc GetTeamMembership (GetTeamMembershipReq) returns (GetTeamMembershipResp) {
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: "teams";
+    };
     option (google.api.http) = {
       get: "/iam/v2/teams/{id}/users"
     };
     option (chef.automate.api.policy).resource = "auth:teams:{id}:users";
     option (chef.automate.api.policy).action = "read";
     option (chef.automate.api.iam.policy).resource = "iam:teams:{id}:users";
     option (chef.automate.api.iam.policy).action = "iam:teamUsers:list";
-  }
+  };
+
+  /*
+  Adds local team membership
+
+  Adds a list of users to a local team. Users are added by their membership_id.
+  The request will ignore any members that do not exist.
+
```

Comment:
Actually, while this request will *functionally* ignore the users that don't exist, the provided ids will still become a part of the teams membership, which is bananas. Another line clarifying that (without belaboring it) would be great.